AlphaWallet/TokenScriptClient/Models/AssetAttributeValueUsableAsFunctionArguments.swift

```diff
@@ -47,6 +47,8 @@ enum AssetAttributeValueUsableAsFunctionArguments {
             return coerceToUInt(forFunctionType: functionType)
         case .void:
             return nil
+        default:
```

Comment:
Needs to coerce them the arguments to the correct type. It might need to be 1 or 2 different types (likely 1), depending on what the function call is used for based on `functionType`. `coerceToAddress` is an example of it being coerced into 2 different types. The reason is because we, unfortunately, use 2 different encoders for calls and transactions.